### PR TITLE
Added org and PID F16A for a DIY throttle controller for flight sims

### PR DESCRIPTION
**Pull request by uriba107**
This has my First attempt at a USB controller, it's currently set to use either ATMEL testing VID/PID or a "never seen" PID under the Thrustmaster LLC VID. which is naturally very bad thing to do. A dedicatedPID for this project will be mostly appreciated. (I have one or two similar projects in the making, so I'll probably submit a request for a PID for them as well once the code will be finalized and commited).

Cheers,
Uri

### CODE REVIEW

File: org/uri_ba/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: org
+title: uri_ba
+site: http://pit.uriba.org/
+---
+I'm a Hobbist, electronics noob which tries his best at building a home cockpit. Which among other things, includes HID controllers for inputs (joysticks and stuff).
```
Comment (Arachnid) [inline org/uri_ba/index.md]: I think you mean 'hobbyist' ;)

File: 1209/F16A/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Cougar TQS adapter
+owner: uri_ba
+license: MIT
```
Comment (Arachnid) [inline 1209/F16A/index.md]: I don't see any indication of this license in your repository - could you add a license file that applies to your software and hardware, please?

Also, your hardware directory includes an Eagle schematic, but not a board file.

Comment (uriba107) [inline 1209/F16A/index.md]: licence is in the project, "licence.txt" in the main folder, and it is applies to the software as it is the original framework license. I've added an additional license file for the HW. and properly indicated it in the project README.

I was not aware that a board design was required, I had only created a schematics as it was intended to be hand soldered. I will create one and commit it later this week.

Thank you,
Uri

**Comment by uriba107** [comment]
I've clarified licensing. and added eagle board file as requested.

File: 1209/F16A/index.md (hunk 2 above)
Comment (uriba107) [inline 1209/F16A/index.md]: @Arachnid 
Licencing was clarified, and Board file added to project as requested.

**Comment by Arachnid** [comment]
Can you fix the typo in the org file, please?

**Comment by uriba107** [comment]
Amended Commit for typo fix.

**Pull request merged**